tests/e2e/blocks/audio-video.spec.js

```diff
@@ -0,0 +1,71 @@
+import {test, expect} from '../tools/lib/test-utils.js';
+import {publishPostAndVisit, createPostWithFeaturedImage} from '../tools/lib/post.js';
+import {searchAndInsertBlock} from '../tools/lib/editor.js';
+
+const YOUTUBE_TEST = 'https://www.youtube.com/watch?v=-CwkccAgKrs';
+const VIMEO_TEST = 'https://vimeo.com/100195272';
+const MP4_TEST = 'https://media.greenpeace.org/AssetLink/60348bthya3s4dg118j4nh42oskv6wdb.mp4';
+
+const SOUNDCLOUD_TEST = 'https://soundcloud.com/greenpeaceuk-1/04-and-we-will-defend-requiem';
+const MP3_TEST = 'https://www.greenpeace.ch/static/planet4-switzerland-stateless/2023/11/b15df1dd-radio_1_wochenserie_greenpeace_231106.mp3';
```

Comment:
My only concern about this consts is that if any external editor removes any of these files, then our checks will fail. If it's possible we can think about uploading with our own. What do you think?

Reply:
Yes that could be possible, I only used what was provided in the ticket description by @comzeradd

Reply:
@comzeradd what do you think?

Reply:
Yes, good point. I uploaded the mp(3|4) files to a bucket we control and updated the urls in the ticket